dashboard/config/initializers/cors.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins CDO.pegasus_site_host
```

Comment:
we'll want to add `hourofocde_site_host` (to be defined) in the future, but that seems fine to leave as future work